src/widgets/index/index.ts

```diff
@@ -171,10 +165,11 @@ const index = (props: IndexProps): Index => {
       // step.
       const mainHelper = instantSearchInstance.mainHelper!;
 
-      const initialSearchParameters =
+      const initialSearchParameters = new algoliasearchHelper.SearchParameters(
```

Comment:
We decided to leave rootPath as is, and use `.make` nowhere in the codebase until it can be proved necessary: https://github.com/algolia/instantsearch.js/issues/2084#issuecomment-512862729